src/LibVLCSharp.WPF/ForegroundWindow.cs

```diff
@@ -142,6 +142,11 @@ void Wndhost_SizeChanged(object? sender, SizeChangedEventArgs e)
 
         void Wndhost_Closing(object? sender, System.ComponentModel.CancelEventArgs e)
         {
+            if (e.Cancel)
```

Comment:
How to test this? How do you make `e.Cancel` be true?

Reply:
It may be assigned from hosting window as below example.

### Example.
```
private void Window_Closing(object sender, System.ComponentModel.CancelEventArgs e)
{
    var result = MessageBox.Show("Are you sure?", "Exit", MessageBoxButton.OKCancel);
        if (result == MessageBoxResult.Cancel)
        {
            e.Cancel = true;
        }
}
```

Reply:
Ok, I see, thanks